### PR TITLE
Remove unused opsbot code

### PR DESCRIPTION
**Pull request by nwalters512**
This PR removes some outdated code and terminology from our opsbot utils.

### CODE REVIEW

File: lib/opsbot.js [hunk 1]
```diff
@@ -84,45 +84,28 @@ module.exports.sendSlackAlmaMessage = (msg, channel, options, callback) => {
   });
 };
 
-module.exports.sendSlackAlmaMessageAsync = util.promisify((msg, channel, options, callback) =>
-  module.exports.sendSlackAlmaMessage(msg, (err, res, body) => callback(err, { res, body }))
```
Comment (nwalters512) [inline lib/opsbot.js:88]: This would have never worked to begin with - it's missing two arguments (channel and options).

File: lib/opsbot.js [hunk 2]
```diff
@@ -47,13 +48,12 @@ module.exports.sendMessageAsync = util.promisify((msg, callback) =>
  * @param callback Function that is called after the message is sent.
  * Called with callback(err, response, body)
  */
-module.exports.sendSlackAlmaMessage = (msg, channel, options, callback) => {
-  // Fall-back to the old token config if it exists
-  const token = config.secretSlackToken || config.secretSlackProctorToken;
+module.exports.sendSlackMessage = (msg, channel, options, callback) => {
```
Comment (nwalters512) [inline lib/opsbot.js:51]: The word "Alma" is a remnant of the early days of this Slack code, when it wrote messages as a user named "Alma". It's not necessary to include that in function names.

File: lib/opsbot.js [hunk 3]
```diff
@@ -84,45 +84,28 @@ module.exports.sendSlackAlmaMessage = (msg, channel, options, callback) => {
   });
 };
 
-module.exports.sendSlackAlmaMessageAsync = util.promisify((msg, channel, options, callback) =>
```
Comment (nwalters512) [inline lib/opsbot.js:87]: All these `*Async` functions are unused.

File: lib/opsbot.js (hunk 1 above)
Comment (nicknytko) [reply lib/opsbot.js:88]: Checked the git blame to see who put this in...

https://pbs.twimg.com/media/FEz4HC5XsAMztm3?format=jpg&name=large

**Review by nicknytko — APPROVED** [review]
looks generally fine to me

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**